src/Plutip/Server.purs

```diff
@@ -12,35 +21,38 @@ import Contract.Monad (Contract, ContractConfig(ContractConfig), runContract)
 import Control.Monad.Error.Class (withResource)
 import Control.Monad.Reader (ReaderT(ReaderT), runReaderT)
 import Data.Bifunctor (lmap)
-import Data.Either (Either(Left), either)
+import Data.Either (Either(..), either, fromRight)
```

Comment:
We use explicit imports everywhere, even for types like Either, please replace `..` with actually needed constructors.